docs/promo.rst

```diff
@@ -0,0 +1,172 @@
+pre-selected for you.) Click *Next* to proceed.
+
+.. image:: _assets/img/free-trial-wizard-step1.png
+   :alt: Promo cluster configuration wizard step 1
+
+In the second step of the configuration wizard, you will be asked to define a
+name for your promo cluster. You will also be asked to define a separate name
+and password to access the cluster directly through the cluster UI, once it is
+deployed. The password must be at least 24 characters. However, do not worry:
+a suitable password will automatically be generated for you.
+
+You can also scale your cluster here between the three available scale units
+of each subscription plan. Note that discounts and free trial offers may apply
+to a particular scale unit only, and may apply for a limited time. The UI will
+show that the relevant scale unit is discounted, by how much, and for how long.
+For example, in the example below, the first scale unit of the Development plan
+is free for one month. When you have filled out the form and scaled as desired,
+click *Next* to proceed.
+
+.. image:: _assets/img/free-trial-wizard-step2.png
+   :alt: Promo cluster configuration wizard step 2
+
+The final step of the configuration wizard simply restates all the information
+about the promo cluster you are about to deploy. You can confirm the desired
+discount here by looking at the Pricing section. When you are satisfied, click
+*Deploy*.
+
+.. image:: _assets/img/free-trial-wizard-step3.png
+   :alt: Promo cluster configuration wizard step 3
+
+If all is well, you will be shown a popup confirming the deployment is in
+progress and showing you once again your cluster username and password. Since
+this is the last time you will be shown the cluster access credentials, you
+may want to copy them into a safe location (e.g., a password manager). Beyond
+the popup, you will see the cluster overview screen in the CrateDB Cloud
+Console. The section *Status* on the left-hand side will show you that the
+cluster is being set up. Once it is done, this will show *Available* and you
+will be able to see cluster metrics.
+
+Naturally, when your free trial or discounted cluster period ends, you can
+continue using it. When required, you can simply add your billing information
+in the Billing tab of the organization overview of the CrateDB Cloud Console.
+(This tab is to the far right of the Subscriptions tab you started at.) For
+more information on using the CrateDB Cloud Console, refer to the `Console
+overview in the reference documentation`_.
+
+
+.. _promo-discounted-cluster:
+
+Deploy a discounted cluster
+===========================
+
+In the case of a discounted - rather than free - cluster, the process is
+essentially the same as described above. You can choose your discounted cluster
+in the subscription plan UI when you have clicked on *Create new subscription*,
+just as before. In the example used here, you could choose the discounted GP
+plan.
```

Comment:
```suggestion
In the case of a discounted - rather than free - cluster, the process is the same
as described above. Choose your discounted cluster in the subscription plan
UI after clicking on *Create new subscription*, just as before. In the example
used here, you could choose the discounted GP plan.
```